src/middleware/validation.middleware.ts

```diff
@@ -55,6 +58,24 @@ export function checkValidations(req: Request, res: Response, next: NextFunction
       }
       const entityNumber = appData?.[EntityNumberKey];
 
+      if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL)) {
+        // This is forthe REDIS removal work, all BO / MO pages need the activeSubmissionBasePath passed into the template
```

Comment:
Minor... 'for the' and 'template constructs'.

Reply:
updated